packages/angular-cli/models/webpack-build-common.ts

```diff
@@ -119,7 +119,13 @@ export function getWebpackCommonConfig(
       new webpack.LoaderOptionsPlugin({
         test: /\.(css|scss|sass|less|styl)$/,
         options: {
-          postcss: [ autoprefixer() ]
+          postcss: [ autoprefixer() ],
+          // workaround for https://github.com/webpack/css-loader/issues/340
+          context: path.resolve(__dirname, './'),
```

Comment:
@hansl Solving this bug is quite important IMHO. Could you please take a closer look and propose an alternative if using `__dirname` is not acceptable?
I've tried the fix and it and works fine (at last).
Thanks a lot!